taskcluster/kinds/tests/kind.yml

```diff
@@ -91,6 +91,20 @@ tasks:
             task lint-black
     run-on-tasks-for: ["github-push", "github-pull-request"]
 
+  lint-pylance:
+    # Run python's pylance type checker.
```

Comment:
> Question

I think maybe this is supposed to be `pright` instead of `pylance`. Is that right?

Reply:
Whoops, yeah pylance is the name of the VSCode extension. That's a mistake.